### ISSUE TITLE
[Feature] Pocket creation for unloading also on counterclockwise headlands

### ISSUE THREAD

**Issue by rohne83** [Courseplay/Courseplay_FS22#2829]
**What is the problem this feature/enhancement solves?**
Combine unloading on headlands is working great already. On clockwise headlands (and in first row of a new land) first combine creates its pocket for unloading, while on later lanes no pocket is required. And even on counterclockwise headlands combines can unload already now, as they drive a bit back and to the side (only on second and following lanes). Only thing missing is pocket creation for also the first combine in counterclockwise headlands.
I use counterclockwise headlands for combine harvest for the first headland quite often. Not only because it's more realistic, but also to avoid the pipe being extended as soon as the combine's trailer trigger detects any trailer at field edge - on fields with obstacles at the edge this can result in problems quite often. But right now there's no possibility to have this combine being unloaded on first headland lane as well (at least not with avoid fruit activated).

**Describe the solution you'd like**
Implement a pocket creation routine also for ccw headlands. See screenshots for how it's done in RL. Quite similar to the clockwise headland's pocket, just that trailer and not the combine will sit in the pocket during unloading.
Not super high priority again, but still a very welcomed feature.

**Describe alternatives you've considered**
N/A

**Additional context**
1. combine gets full and backs up:
![fsScreen_2023_10_03_09_57_53](https://github.com/Courseplay/Courseplay_FS22/assets/71030138/2de55045-0d86-460f-b44b-d01bdb298527)
2. combine creates pocket:
![fsScreen_2023_10_03_09_58_11](https://github.com/Courseplay/Courseplay_FS22/assets/71030138/b3e8725d-0f7d-4ad3-a19a-34a7175015ab)
3. combine drives back to its lane, unloader approaching combine for unload
![fsScreen_2023_10_03_09_58_59](https://github.com/Courseplay/Courseplay_FS22/assets/71030138/9e46a393-7020-46c2-93f9-b52c7c65735f)

**Comment by Tensuko** [comment]
If so, then only on 1st headland.
Else we could never decide if reverse out of fruit or pocket.....
But I don't see any reason why it should drive CCW.
And no field entrance doesn't count, there is always a way to get into the work in CW on an entrance.

**Comment by rohne83** [comment]
of course only on first headland. All other situations are handled correctly already now.
Don't understand what you mean with field entrance though

**Comment by Tensuko** [comment]
Feldeinfahrt wurde mal als Grund genannt, warum man gegen den Uhrzeigersinn im Headland fährt.
Das sehe ich eben nicht so.
Ich kenne keinen Grund, warum man nicht um Uhrzeigersinn fahren sollte, wenn das Rohr auf der linken Seite ist um eben genau dieses Fahrmanöver nicht haben zu müssen.

**Comment by Tensuko** [comment]
Ok verstehe. Langes Rohr schmale Arbeitsbreite das ist nachvollziehbar.
Danke.

**Comment by pvaiko** [comment]
> Ok verstehe. Langes Rohr schmale Arbeitsbreite das ist nachvollziehbar. Danke.

Does not mall working width and long pipe mean the pocket must be cut wider then the working width? Making this maneuver more complicated?

**Comment by rohne83** [comment]
Indeed, but would say that's a different topic and independent of the headland direction. When I think of some sugarbeet/potato harvesters you have the issue with too narrow pockets also for the normal pockets on clockwise headlands. So that would need a dedicated solution approach which then can be applied on existing and the new pocket creation maneuvers.

**Comment by blubstar86** [comment]
> Ok verstehe. Langes Rohr schmale Arbeitsbreite das ist nachvollziehbar. Danke.

Im RL wurde auch mal genannt, dass es teilweise drauf an kommt auf welcher Seite der Antrieb fürs Schneidwerk ist, da diese Seite dann breiter ist und weiter aus dem Feld herausragen würde und evtl. an Dingen die sehr nah Feldrand sind (Zaun, Baum etc. ) eher hängen bleiben könnte
